README.md

```diff
@@ -219,8 +219,8 @@ $ docker run --rm \
 ## Development
 
 [embedmd]:# (tmp/make_help.txt)
```

Comment:
@kakkoyun what does this do? I wonder if this has to do with the extra newline that I am seeing. ahead of this code block.

Reply:
Probably that's the reason. This an annotation for a tiny program (https://github.com/campoy/embedmd) that we run to embed code pieces or generated things into markdown files. It's quite handy. Especially for the dynamic content. You can find the detailed usage in the Makefile. https://github.com/meltwater/drone-cache/blob/3172b74d6b3e9c1905b0a18f7fcb69604483609b/Makefile#L66-L68

Reply:
I think if you remove leading `\n` in `printf` statement, it should work.

Reply:
I see. It doesn't look like we are generating any dynamic help text here at all, are we?

Reply:
I removed the `\n` now as you suggested. 

However I don't understand how this is working.
I ran `make README.md` however that lead to a duplicate code block in the `## Development` section.

You see me a bit confused :)